src/main/java/org/jabref/logic/citationstyle/CitationStyle.java

```diff
@@ -71,7 +71,9 @@ private static CitationStyle createCitationStyleFromSource(final String source,
                 String title = ((CharacterData) titleNode.item(0).getFirstChild()).getData();
 
                 return new CitationStyle(filename, title, source);
-            } catch (ParserConfigurationException | SAXException | IOException e) {
+            } catch (SAXException ignore) {
+                // is triggered when a use has not configured CitationStyles -> ignore
```

Comment:
Is there no easy way to prevent the exception from being triggered when the user actually does not use citation styles (I suspect these null checks in the beginning should actually serve this job, although they fail obviously). Reason: I think these SAX errors also appears if a user uses a citation style but the parsing fails for some reason. In this case the logged error message is actually helpful.

If there is no easy way to prevent the exception, then I'm fine with merging this PR.

Reply:
You are right of course, I should rather track down the origin. The problem lies in the citationstyles. The first line of some of the styles is broken. For example, the style `university-of-york-vancouver.csl` looks like this `?<?xml version="1.0" encoding="utf-8"?>` and is invalid because it starts with a question mark. 

That should be fixed at the end of the citation styles. For now, I have pushed a fix on our side that checks if the first character of the citation style is also the expected `<` and avoids parsing the style if it is not.